src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
```

Comment:
Sorry, that's a habit. In BlackCat CMS, especially in helper classes, all methods are declared as static but can be called in object context, too.